### ISSUE TITLE
Possibility for native animations?

### ISSUE THREAD

**Issue by Dyphen** [sdias/win-10-virtual-desktop-enhancer#35]
I know it's silly, but I was wondering how feasible it is to use the native "slide" animation when switching between virtual desktops.

**Comment by sdias** [comment]
Hi. I don't think it's silly, it could probably be done. Personally, I wouldn't use it because I find it a bit confusing and I like it to be instant. Would you?

**Comment by aelfredwulf** [comment]
Yes! Please enable the default slide animation.

**Comment by moymike** [comment]
The problem with using the slide animation, is that the wallpaper won't "slide" with it.  The result is the desktop will slide over until on the new screen, then the wallpaper will swap out.  The current, no slide, switches the desktop with the new wallpaper without the animation. 

Using the standard windows shortcut to "slide" the desktop while the enhancer was running, allows you to see how the wallpaper switch after the screen slides in looks.  Personally, it sucks.  Much nicer to pop it in with the new wallpaper then slide it over then pop the wallpaper in.

Hopefully Microsoft will add the ability to have different wallpaper per virtual desktop and "slide" it in.  (Mac Book does this)

**Comment by sdias** [comment]
Yeah... My app tries to overcome some limitations using workarounds (like re-setting your wallpaper on-the-fly as you change desktops), but you can only do so much with what is available.

For example, the only way I'd be able to implement this feature (using the slide animation for any keyboard shortcut) would be for me to replicate pressing `Win+Ctrl+Left/Right` as many times as necessary to navigate to the right desktop.

I'll do some testing to see if it is feasible.

**Comment by ariofrio** [comment]
I don't use the different wallpapers feature, so being able to use the native animations even if different wallpapers per virtual desktop doesn't work correctly, would work just fine for me. I imagine other users might have the same experience. Great job on this, btw!

**Comment by dracan** [comment]
I'm with @ariofrio. I've just started using this as I really wanted a hotkey to be able to move windows (as he said - great job!). I don't use or need different wallpaper, but do really miss the scroll animation. It's a shame that the multi-wallpaper functionality that perhaps a lot of people don't even use, is preventing the native animation from working.

**Comment by dracan** [comment]
Actually, I've just noticed the `UseNativePrevNextDesktopSwitchingIfConflicting` setting. This brings back the scroll, and I can still use the *move window to desktop* shortcuts! Perfect! 🎉

**Comment by kkuhle** [comment]
@dracan Thanks for pointing this out!  Came here looking for an answer to this
